movie-search.js

```diff
@@ -18,9 +21,10 @@ const printTitles = (rawData) => {
   titles.forEach((element) => {
     console.log(element)
   })
+  return titles
```

Comment:
Ahh, good catch. The second way is better. foreach actually passes 3 elements to it's callback, the current element, the index, and the entire array. console.log takes any number of arguments and then prints them all. So what I did is the same as this:

```js
titles.foreach((i, element, array) => console.log(i, element, array))
```

What you did is better.